src/app/loaders/express/helmet.ts

```diff
@@ -1,6 +1,6 @@
 import { RequestHandler } from 'express'
 import helmet from 'helmet'
-import { ContentSecurityPolicyOptions } from 'helmet/dist/middlewares/content-security-policy'
+import { ContentSecurityPolicyOptions } from 'helmet/dist/types/middlewares/content-security-policy'
```

Comment:
I merged latest develop leh; is it in a hotfix :O... ahhhhhhhhhh because of https://github.com/opengovsg/FormSG/pull/3433. Can we merge that in? I'll update it after that's merged